### ISSUE TITLE
Removing cookies library as a dependency

### ISSUE THREAD

**Issue by dhuang** [spulec/moto#1812]
I hit this deprecation warning today:

```
/usr/src/.tox/py36/lib/python3.6/site-packages/cookies.py:312: DeprecationWarning: Flags not at the start of the expression '(?ix)  # Case-insens' (truncated)                                                                                                                                                               
  ATTR_RE = re.compile(ATTR)  
```

It is due to the cookies library that moto depends on, see https://gitlab.com/sashahart/cookies/issues/4. There is no fix yet and this will be a problem for Python 3.7 compatibility. However, I was not able to find where moto actually uses this library and was wondering if I could just remove it from setup.py entirely.

**Comment by dhuang** [comment]
Got it, thanks for confirming! Opened https://github.com/spulec/moto/pull/1835.

**Comment by spulec** [comment]
Closed with #1835
